pkgs/development/python-modules/urllib3/default.nix

```
@@ -23,7 +23,8 @@ buildPythonPackage rec {
   doCheck = false;
 
   checkInputs = [ pytest mock tornado ];
-  propagatedBuildInputs = [ pyopenssl cryptography idna certifi ipaddress pysocks ];
+  propagatedBuildInputs = [ ipaddress pysocks ]
```

```suggestion
  propagatedBuildInputs = [ pysocks ]
```
https://github.com/urllib3/urllib3/blob/1.26.3/setup.py#L93

---

Good catch. Thanks!